### DISCUSSION TITLE
implement github project code and publish for free

### DISCUSSION THREAD

**Discussion by Ashwiin** [community/community#21399] [Other Feature Feedback, Questions, & Ideas]
<p>Hi Guys, I am new here.</p>
<p>Just wanted to know, am I allowed to publish a code for free after getting a project from GitHub community and implementing it to a part of my own coded project and upload to the store for free? Am I wrong or is it ok to do this?</p>
<p>I would like to upload my first app idea and would need a project in GitHub to be implemented in my code as a part. Please advise me on this. Greatly appreciated :) </p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/ashwiin">@ashwiin</a>,</p>
<p>Welcome and thank you for being here! You can share your projects here for feedback from the community, for tips on how to get started with GitHub check out this <a href="https://github.community/t/new-to-everything/1526/4">thread from the community</a>.</p>
<p>Good Luck!</p>
<p>Andrea</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/ashwiin">@ashwiin</a>,</p>
<p>It sounds like you’re asking about license restrictions. Each project has a license associated with it. Those licenses dictate how projects can be used and what sort of requirements they have for being included in other projects. There is no standard license for projects uploaded to GitHub. You will need to check the license of each project you use to see if you can use and distribute it in your own project.</p>
<p>Here is an <a href="https://choosealicense.com/licenses/" rel="nofollow noopener">overview of the most common licenses</a> and what they mean.</p>
<p>Hope this helps!</p>
